File: VB/CustomSplashForm.Win/SplashScreenForm.vb
Imports System.ComponentModel
Imports System.Linq
Imports System.Text

Namespace CustomSplashForm.Win
    Partial Public Class SplashScreenForm
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub
        Friend Sub UpdateInfo(ByVal info As String)
            label2.Text = info
        End Sub
    End Class
End Namespace

File: CS/CustomSplashForm.Win/SplashScreenForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace CustomSplashForm.Win {
    public partial class SplashScreenForm : Form {
        public SplashScreenForm() {
            InitializeComponent();
        }
        internal void UpdateInfo(string info) {
            label2.Text = info;
        }
    }
}
